app/serializers/forest_liana/stripe_invoice_serializer.rb

```diff
@@ -3,20 +3,20 @@ class StripeInvoiceSerializer
     include JSONAPI::Serializer
 
     attribute :amount_due
+    attribute :amount_paid
```

Comment:
Doesn't those depends on the stripe version ? What if someone use on old version ofStripe ?